events/app.py

```diff
@@ -47,7 +47,7 @@ def add_event():
             response="Event info was entered incorrectly.",
         )
     try:
-        event.add_to_db(EVENTS_COLL)
+        add_to_db(event.dict, EVENTS_COLL)
```

Comment:
Given the availability of mongomock, why not have this just call `EVENTS_COLL.insert_one(event.dict)`?

Reply:
Thanks - I didn't realize that it was simpler to just add to the DB directly there.